apis/v1beta1/httproute_types.go

```diff
@@ -901,7 +902,8 @@ type HTTPRequestRedirectFilter struct {
 
 	// Port is the port to be used in the value of the `Location`
 	// header in the response.
-	// When empty, port (if specified) of the request is used.
+	// When empty, the Gateway listener port is used (except
+	// when it is a default port - 80 or 443).
```

Comment:
I don't think this "except ..." part is clear. I actually don't know what it means

Reply:
I was trying to say this in a succinct way - When the Port is unspecified, The port from the Gateway Listener will be appended to the Location Header, except when the Gateway Listener port is a default/standard port (80 for HTTP and 443 for HTTPS ).
Any suggestions here ?

Reply:
I'd say something like:
```
In all cases, when the protocol is http and the port is port 80, then the port must be omitted.
Similarly, when the protocol is https and the port is port 443, then the port must also be omitted.
```

Reply:
hey @youngnick are you suggesting that even when the redirect port is explicitly set to `80` by the user, it is not appended to the `Location` header ?

Reply:
Yes, the Location header should, like all URLs, only show the port when it's relevant, which is when it's _not_ 80 for http, or 443 for https.

So if a request comes in on `http://foo.bar.com:8080/baz` and is rewritten to `http://quux.com/baz`, then you can tell the port is 80 because it's not present.

Reply:
thanks for clarification, will update the PR !